actions/feedback_actions.py

```diff
@@ -49,6 +52,7 @@
         FuzzySlotMatchOption(
             "general",
             [
+                "no",
```

Comment:
Could be easier if we detect `intent_core_no` before checking the fuzzy slot match